go/libraries/doltcore/sqle/dfunctions/hashof.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Dolthub, Inc.
+// Copyright 2021 Dolthub, Inc.
```

Comment:
You can go ahead and extend the copyright year range: 
```suggestion
// Copyright 2020-2023 Dolthub, Inc.
```